backend/application/resources/user.py

```diff
@@ -0,0 +1,10 @@
+from apiflask import APIBlueprint, abort
+
+from infrastructure.database.entities import User
+
+router = APIBlueprint("transaction", __name__)
+
+@router.get('/user/<id>')
+def get_user(id):
+    user = User.query.filter_by(id=id).first()
+    return {"name": user.name}
```

Comment:
Para retornar o usuário esquematizado você pode criar um schema em vez de fazer dessa forma. Assim não vai precisar criar um objeto toda vez que for precisar retornar.